src/init.cpp

```diff
@@ -994,6 +995,8 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     if (GetBoolArg("-peerbloomfilters", true))
         nLocalServices |= NODE_BLOOM;
 
+    nMaxTipAge = GetArg("-maxtipage", DEFAULT_MAX_TIP_AGE);
```

Comment:
I've always tried to avoid using the current value as default anywhere for GetArg. Specifying DEFAULT_MAX_TIP_AGE is more clear when reading the code, and when at some point argument parsing is refactored it's nice to have the defaults on hand without having to 'reason back' about the value.